dashboard2/settings/jptt.md

```diff
@@ -179,6 +179,28 @@ could be expressed as
 let publishedText = `Published on ${date}`;
 ```
 
-Simple cases like the above (or literals without variables) are supported. The expression will become a tag in the Workbench and in XLIFF exports. JavaScript allows any expression to be added to literals but we recommend you to stick to simple expressions if possible.
+Cases like the above (or literals without variables) are now supported. The expression will become a tag in the Workbench and in XLIFF exports. 
```

Comment:
An `x` tag, right? It'll appear as a single opaque element, and not in a paired `g` tag.